### ISSUE TITLE
ui: /ts/query fails on recent build

### ISSUE THREAD

**Issue by mberhault** [cockroachdb/cockroach#6626]
Just upgraded `gamma` and `rho` to 5dde710a08b721ca29aa78d93a31621283503f21
Previous build was: 4837761

All graphs are broken with the following response to the `/ts/query` requests:
```
json: invalid use of ,string struct tag, trying to unmarshal unquoted value into int64
```

**Comment by mberhault** [comment]
tried clearing the cache, no luck.

**Comment by mberhault** [comment]
@mrtracy: https://github.com/cockroachdb/cockroach/commit/59e803353c068a04b5b8ba2f415bd2a00727358c I'm guessing?

**Comment by mrtracy** [comment]
Fixed by #6635
